cra-rxjs-styled-components/src/components/issue/IssueCard/IssueCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ClosedIssueIcon from '../../icons/ClosedIssueIcon';
+import MessageIcon from '../../icons/MessageIcon';
+import OpenIssueIcon from '../../icons/OpenIssueIcon';
+import PullRequestIssueInfo from '../../PullRequestIssueInfo';
+import { IssueCardWrapper } from './IssueCard.style';
+import type { Status } from '../types';
+
+interface Props {
+  status: Status;
+  title: string;
+  openedNum: string;
+  openedDay: string;
+  openedBy: string;
+  messageCount: number;
+}
+
+export default function IssueCard({
+  status,
+  title,
+  openedBy,
+  openedDay,
+  openedNum,
+  messageCount,
+}: Props) {
+  const getPRIcon = (status: Status) => {
+    switch (status) {
+      case 'closed':
+        return <ClosedIssueIcon />;
+      case 'open':
+      default:
+        return <OpenIssueIcon />;
+    }
+  };
```

Comment:
Would a ternary not have worked here? It's not a problem I'm just curious about the choice of a switch vs ternary or if/else.

Reply:
It would have, this seems neater. Or what do you think?

Reply:
I think because there are only 2 cases, I'd go with the if/else. I don't like ternaries for this type of thing personally to throw my 2¢ into the convo.

I personally reserve switch statements when there are 4+ cases I'm trying to manage.